site/man/rabbitmq-streams.8.html

```diff
@@ -232,6 +232,16 @@ <h3 class="Ss" title="Ss" id="Stream_plugin"><a class="permalink" href="#Stream_
       <var class="Ar" title="Ar">consumerinfoitem</var> can take any value from
       the list that follows:
     <dl class="Bl-tag">
+      <dt><var class="Ar" title="Ar">active</var></dt>
+      <dd>Boolean indicating whether the consumer is active or not.</dd>
```

Comment:
Is it a boolean? I thought booleans took only true or false values? Is active = true?